### PR TITLE
Contact me section -pair programming-

### PR DESCRIPTION
**Pull request by uuukiiiyooo**
This update adds the "Contact me" section at the bottom of the mobile version of the home page, it also features:

- A contact form with three required inputs
- A submit button that sends collected information to Formspree service
- Different commits made by my coding partner and myself

### CODE REVIEW

**Review by NahnahAJ — COMMENTED** [review]
# STATUS: INVALID!!!  :x: :x: :x: 

Hi @uuukiiiyooo and @naanmohammed  ,

While you made a great effort in this project, unfortunately, I cannot proceed to review your code, please make sure you implement the following changes before you resubmit for review. 

## Required Changes ♻️
_Check the comments below._
- [ ] Please you have submitted a project that does not have a professional readme file, check out this [template ](https://github.com/microverseinc/readme-template)from Microverse and adapt it to your project. Please ensure all the sections and subsections of the template are updated with information pertaining to you or your project. 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear or you can get to me on slack using this handle @nahnahAJ.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.html [hunk 1]
```diff
@@ -198,5 +198,23 @@ <h2 class="aboutLink">LET'S CONNECT</h2>
...
+                <li>
+                    <input name="user_name" id="name" placeholder="Yeremias" type="text" maxlength="30" required>
+                </li>
+                <li><input name="email" id="email" placeholder="natajayanj14@gmail.com" type="email" required>
+                </li>
+                <li>
+                    <textarea name="comment" id="comment" placeholder="Write your message here" maxlength="500" required></textarea>
+                </li>
+                <button type="submit">Get in touch</button>
+            </ul>
```
Comment (ouasamine) [inline index.html:216]: - [ ] Nice work implementing the submit button successfully, however, you still need to set the proper interactions for this button as shown in the Figma design 🔽 remember that according to the project's [requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/3_contact_form.md), you need to stick to the Figma design as much as possible.

File: index.html [hunk 2]
```diff
@@ -198,5 +198,23 @@ <h2 class="aboutLink">LET'S CONNECT</h2>
...
+        <h2>Contact me</h2>
+        <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
+        <form action="https://formspree.io/f/mbjeoppw" method="post">
+            <ul>
+                <li>
+                    <input name="user_name" id="name" placeholder="Yeremias" type="text" maxlength="30" required>
+                </li>
+                <li><input name="email" id="email" placeholder="natajayanj14@gmail.com" type="email" required>
+                </li>
+                <li>
```
Comment (ouasamine) [inline index.html:212]: - [ ] This is an example from your `index.html` where the used indentation(the space preceding each new line of code) is set to 4 spaces, which is not a good practice, please make sure to use 2 spaces for your default indentation as it improves readability, please refer to this [lesson](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md) for more info.

**Review by ouasamine — CHANGES_REQUESTED** [review]
Hi @uuukiiiyooo 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
- Implemented all core functions successfully ✔️ 
- Nice work on the UI generally 👏🏽 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (uuukiiiyooo) [reply index.html:216]: Thanks for your kind feedback!

**Review by vickymarz — APPROVED** [review]
Hi @uuukiiiyooo, 

# ✔️   Status: APPROVED 🏆 🎉 🎊 

You have done a great job on this project, keep it up 👍 🥇. Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
congratulations!  🎉💯🌟 

## Top highlight 🎯

- The Pull Request is following the GitHub Flow ✅ 
- The PR title and description are professional and descriptive ✅ 



## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**